plugins.json

```diff
@@ -1832,6 +1832,50 @@
     "enterprise": true,
     "updated_at": "2020-07-10T19:29:56.989149472Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-nps",
+    "icon_data": "data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNTAwIiBoZWlnaHQ9IjUwMCIgdmlld0JveD0iMCAwIDUwMCA1MDAiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMzk2LjkgNDcuNzAwMUwzOTkuNSAxMDAuOEM0NDIuNSAxNDguMyA0NTkuNSAyMTUuNiA0MzguMSAyNzguOUM0MDYuMSAzNzMuMyAzMDAuNyA0MjMgMjAyLjcgMzg5LjhDMTA0LjcgMzU2LjYgNTEuMTAwMiAyNTMuMSA4My4wMDAyIDE1OC43QzEwNC41IDk1LjIwMDEgMTU5LjIgNTIuMDAwMSAyMjIuNSA0MC41MDAxTDI1Ni43IDAuMTAwMDk0QzE1MCAtMi43OTk5MSA0OS4zMDAyIDYzLjQwMDEgMTMuMzAwMiAxNjkuOUMtMzAuOTk5OCAzMDAuNiAzOS4xMDAyIDQ0Mi41IDE2OS45IDQ4Ni43QzMwMC43IDUzMC45IDQ0Mi41IDQ2MC45IDQ4Ni44IDMzMC4xQzUyMi43IDIyMy45IDQ4My4xIDExMC4zIDM5Ni45IDQ3LjcwMDFaIiBmaWxsPSIjMDA1OENDIi8+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNMzM1LjYgMjA0LjNMMzMzLjggMTMwLjFMMzMyLjMgODcuNDAwMkwzMzEuMyA1MC40MDAyQzMzMS4zIDUwLjQwMDIgMzMxLjUgMzIuNjAwMiAzMzAuOSAyOC40MDAyQzMzMC44IDI3LjUwMDIgMzMwLjUgMjYuODAwMiAzMzAuMiAyNi4yMDAyQzMzMC4yIDI2LjEwMDIgMzMwLjEgMjYuMDAwMiAzMzAuMSAyNS45MDAyQzMzMC4xIDI1LjgwMDIgMzMwIDI1LjcwMDIgMzMwIDI1LjcwMDJDMzI5LjMgMjQuNTAwMiAzMjguMiAyMy42MDAyIDMyNi45IDIzLjEwMDJDMzI1LjUgMjIuNjAwMiAzMjQgMjIuNzAwMiAzMjIuNyAyMy4zMDAySDMyMi42QzMyMi40IDIzLjQwMDIgMzIyLjMgMjMuNDAwMiAzMjIuMiAyMy41MDAyQzMyMS42IDIzLjgwMDIgMzIxIDI0LjIwMDIgMzIwLjQgMjQuODAwMkMzMTcuNCAyNy44MDAyIDMwNi43IDQyLjAwMDIgMzA2LjcgNDIuMDAwMkwyODMuNSA3MC44MDAyTDI1Ni40IDEwMy44TDIwOS45IDE2MS42QzIwOS45IDE2MS42IDE4OC42IDE4OC4yIDE5My4zIDIyMUMxOTggMjUzLjggMjIyLjQgMjY5LjcgMjQxLjMgMjc2LjFDMjYwLjIgMjgyLjUgMjg5LjMgMjg0LjYgMzEyLjkgMjYxLjRDMzM2LjQgMjM4LjQgMzM1LjYgMjA0LjMgMzM1LjYgMjA0LjNaIiBmaWxsPSIjMDA1OENDIi8+Cjwvc3ZnPgo=",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-nps-v1.1.0.tar.gz",
+    "release_notes_url": "",
```

Comment:
Missing this URL. I suppose it is not important.

Reply:
Given that we are bundling NPS with 5.28 it doesn't seem important. I will still add it for the next release.